### DISCUSSION TITLE
openvpn-server编译出错

### DISCUSSION THREAD

**Discussion by ultra01** [immortalwrt/immortalwrt#1607] [General]
ERROR: luci-app-openvpn-server-3.0-r0: trying to overwrite etc/config/openvpn owned by openvpn-openssl-2.6.12-r1.
ERROR: luci-app-openvpn-server-3.0-r0: trying to overwrite etc/easy-rsa/vars owned by openvpn-easy-rsa-3.2.1-r1.

**Comment by GreatMichaelLee** [comment]
同样的问题
